src/components/tool/index.tsx

```diff
@@ -0,0 +1,123 @@
+import styled from "styled-components";
+import { useClient, useSchema } from "sanity";
+
+import schemaTypesToViewData from "./schema-types-to-view-data";
+import _ from "lodash";
```

Comment:
bunch of regular lodash you may not have seen, can probably run the codemod to get most of it